lib/matplotlib/font_manager.py

```diff
@@ -1,7 +1,7 @@
 """
 A module for finding, managing, and using fonts across platforms.
 
-This module provides a single `FontManager` instance that can
+This module provides a single `FontManager` instance, ``fontManager``, that can
```

Comment:
I did not manage to link to `fontManager` when the definition is in `font_manager_api.rst`.